src/pyOpenMS/pxds/RNPxlDeisotoper.pxd

```diff
@@ -9,5 +9,14 @@ cdef extern from "<OpenMS/ANALYSIS/RNPXL/RNPxlDeisotoper.h>" namespace "OpenMS":
 
 # COMMENT: wrap static methods
 cdef extern from "<OpenMS/ANALYSIS/RNPXL/RNPxlDeisotoper.h>" namespace "OpenMS::Deisotoper":
-        void deisotopeAndSingleCharge(MSSpectrum & in, double fragment_tolerance, bool fragment_unit_ppm, int min_charge, int max_charge, bool keep_only_deisotoped, unsigned int min_isopeaks,  unsigned int max_isopeaks, bool make_single_charged, bool annotate_charge) nogil except + # wrap-attach:Deisotoper
+        void deisotopeAndSingleCharge(MSSpectrum & sepctra, 
```

Comment:
@timosachsenberg actually it would have been sufficient to only change the pxd :-) but its better and clearer that way

Reply:
Except for the typo;)